.github/workflows/node.js.yml

```
@@ -26,6 +26,7 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
         cache: 'npm'
+    - run: npm install
```

This will do an install based the `package-lock.json` without modifying it instead of the `package.json` too.
```suggestion
    - run: npm ci
```